specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -6193,6 +6462,41 @@
       },
       "x-ms-discriminator-value": "Email"
     },
+    "WebhookHookParameterPatch": {
+      "required": [
+        "endpoint"
+      ],
+      "type": "object",
+      "properties": {
+        "endpoint": {
+          "description": "API address, will be called when alert is triggered, only support POST method via SSL",
+          "type": "string"
+        },
+        "username": {
+          "description": "The password, if using basic authentication",
```

Comment:
I think description should be "The **username**, if using basic authentication".

Reply:
Fixed. Thanks for catching.